socialhome/streams/app/tests/main.tests.js

```diff
@@ -0,0 +1,8 @@
+import {should} from "chai"
+should()
```

Comment:
Got from a quick read of  [the documentation](http://chaijs.com/guide/styles/#should). Did I get this wrong?